src/shared/components/headerClipMaskRight/headerClipMaskRight.css

```diff
@@ -0,0 +1,7 @@
+.mask {
```

Comment:
Seems as though between 700px and 800px the clip path creates space between the browser frame edge and the image edge.

Check this out, btw: https://stackoverflow.com/questions/34354770/curved-line-at-the-bottom-of-border-in-css